tests/MongoDB.Driver.Tests/Specifications/change-streams/ChangeStreamTestRunner.cs

```diff
@@ -304,7 +328,8 @@ private void AssertEvents(List<CommandStartedEvent> actualEvents, List<BsonDocum
 
             if (actualEvents.Count > expectedEvents.Count)
             {
-                throw new Exception($"Unexpected command started event: {actualEvents[n].CommandName}.");
+                // the tests assume that a number of actual events can be bigger than expected.
```

Comment:
The newly added test expects only one `aggregate` event. However, we have 3. 
Discussed with @jyemin, we need to be sure that only expected events were triggered. All additional events should be skipped. See, how it's implemented in java: https://github.com/mongodb/mongo-java-driver/blob/f0ad2a2df5b418d269235cbe7183cf9d009dea76/driver-sync/src/test/functional/com/mongodb/client/ChangeStreamsTest.java#L182-L195

Reply:
Acknowledged.  There are actually many command started events that are generated here between the multiple aggregate commands, the failed getMore commands, and the killCursors commands before each resume.

Reply:
How do you know that extra actual events should be ignored for ALL tests and not just SOME tests?

Reply:
I don't know. I thought about an array with test names which should be ignored here. But since java testrunner doesn't worry about it, I just copied their solution. If you think that we need to add it, I don't mind.

Reply:
I still don't understand why we can get away with ignoring extra events that we weren't expecting.

But since Jeff says it's OK I will rely on that and LGTM this pull request.